code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -819,9 +819,6 @@ var/list/robot_verbs_default = list(
 		return ..()
 
 /mob/living/silicon/robot/emag_act(user as mob)
-	if(!ishuman(user))

```

Comment:
You're right. For some reason I was thinking they should only be able to unlock covers, not subvert other borgs.